lib/ansible/modules/cloud/amazon/batch_compute_environment.py

```diff
@@ -0,0 +1,496 @@
+
+            # if region is not provided, then get default profile/session region
+            if not self.region:
+                self.region = self.resource_client['batch'].meta.region_name
+
+        except (ClientError, ParamValidationError, MissingParametersError) as e:
+            ansible_obj.fail_json(msg="Unable to connect, authorize or access resource: {0}".format(e))
+
+        # set account ID
+        try:
+            self.account_id = self.resource_client['iam'].get_user()['User']['Arn'].split(':')[4]
+        except (ClientError, ValueError, KeyError, IndexError):
+            self.account_id = ''
+
+    def client(self, resource='batch'):
+        return self.resource_client[resource]
+
+
+def cc(key):
+    """
+    Changes python key into Camel case equivalent. For example, 'compute_environment_name' becomes
+    'computeEnvironmentName'.
+
+    :param key:
+    :return:
+    """
+    components = key.split('_')
+    return components[0] + "".join([token.capitalize() for token in components[1:]])
+
+
+def set_api_params(module, module_params):
+    """
+    Sets module parameters to those expected by the boto3 API.
+
+    :param module:
+    :param module_params:
+    :return:
+    """
+
+    api_params = dict()
+
+    for param in module_params:
+        module_param = module.params.get(param, None)
+        if module_param is not None:
+            api_params[cc(param)] = module_param
+    return api_params
+
+
+def validate_params(module, aws):
+    """
+    Performs basic parameter validation.
+
+    :param module:
+    :param aws:
+    :return:
+    """
+
+    compute_environment_name = module.params['compute_environment_name']
+
+    # validate compute environment name
+    if not re.search('^[\w\_:]+$', compute_environment_name):
+        module.fail_json(
+            msg="Function compute_environment_name {0} is invalid. Names must contain only alphanumeric characters "
+                "and underscores.".format(compute_environment_name)
+        )
+    if not compute_environment_name.startswith('arn:aws:batch:'):
+        if len(compute_environment_name) > 128:
+            module.fail_json(msg='compute_environment_name "{0}" exceeds 128 character limit'
+                             .format(compute_environment_name))
+
+    return
+
+
+# ---------------------------------------------------------------------------------------------------
+#
+#   Batch Compute Environment functions
+#
+# ---------------------------------------------------------------------------------------------------
+
+def get_current_compute_environment(module, connection):
+    try:
+        environments = connection.client().describe_compute_environments(
+            computeEnvironments=[module.params['compute_environment_name']]
+        )
+        return environments['computeEnvironments'][0] if len(environments['computeEnvironments']) > 0 else None
```

Comment:
Instead of one-line `if` can you use a standard `if/else` here?

Reply:
fixed